### DISCUSSION TITLE
node/node%E6%96%87%E6%A1%A3/Corepack

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#884] [Announcements]
# node/node%E6%96%87%E6%A1%A3/Corepack

Corepack Corepack 是 Node.js 的一个实验性工具，它用于管理 JavaScript 包管理器的版本。JavaScript 包管理器，比如 npm、Yarn 或 pnpm，是用来帮助开发者安装和管理项目依赖的工具。 在 Node.js 中，npm 通常会预装。但是，如果你想使用不同版本的 npm，或者其他包管理器如 Yarn 和 ...

https://doc.cherrychat.org/node/node%E6%96%87%E6%A1%A3/Corepack.html

**Comment by ruan-cat** [comment]
谢谢博主。 `corepack prepare pnpm@9.7.0 --activate` 有效解决了我升级pnpm的问题。
